src_tests/unittests/osm_configurator/view/states/test_positioned_frame.py

```diff
@@ -0,0 +1,106 @@
+                           "NSEW"),
+                          (top_level_frame_stub_i.TopLevelFrameStub(None),
+                           positioned_frame_i.SMALLEST_COLUM_AND_ROW_VALUE,
+                           positioned_frame_i.SMALLEST_COLUM_AND_ROW_VALUE,
+                           positioned_frame_i.SMALLEST_COLUM_SPAN_AND_ROW_SPAN_VALUE,
+                           positioned_frame_i.SMALLEST_COLUM_SPAN_AND_ROW_SPAN_VALUE - 1,
+                           "NSEW")
+                          ])
+def test_value_error(top_level_frame, colum, row, colum_span, row_span, sticky):
+    with pytest.raises(ValueError):
+        positioned_frame = positioned_frame_i.PositionedFrame(
+            frame=top_level_frame, colum=colum, row=row, colum_span=colum_span, row_span=row_span, sticky=sticky)
+
+
+@pytest.mark.parametrize("top_level_frame,colum,row,colum_span,row_span,sticky",
+                         [(top_level_frame_stub_i.TopLevelFrameStub(None),
+                           "Something",
+                           positioned_frame_i.SMALLEST_COLUM_AND_ROW_VALUE,
+                           positioned_frame_i.SMALLEST_COLUM_SPAN_AND_ROW_SPAN_VALUE,
+                           positioned_frame_i.SMALLEST_COLUM_SPAN_AND_ROW_SPAN_VALUE,
+                           "NSEW"),
+                          (top_level_frame_stub_i.TopLevelFrameStub(None),
+                           positioned_frame_i.SMALLEST_COLUM_AND_ROW_VALUE,
+                           "Something",
+                           positioned_frame_i.SMALLEST_COLUM_SPAN_AND_ROW_SPAN_VALUE,
+                           positioned_frame_i.SMALLEST_COLUM_SPAN_AND_ROW_SPAN_VALUE,
+                           "NSEW"),
+                          (top_level_frame_stub_i.TopLevelFrameStub(None),
+                           positioned_frame_i.SMALLEST_COLUM_AND_ROW_VALUE,
+                           positioned_frame_i.SMALLEST_COLUM_AND_ROW_VALUE,
+                           "Something",
+                           positioned_frame_i.SMALLEST_COLUM_SPAN_AND_ROW_SPAN_VALUE,
+                           "NSEW"),
+                          (top_level_frame_stub_i.TopLevelFrameStub(None),
+                           positioned_frame_i.SMALLEST_COLUM_AND_ROW_VALUE,
+                           positioned_frame_i.SMALLEST_COLUM_AND_ROW_VALUE,
+                           positioned_frame_i.SMALLEST_COLUM_SPAN_AND_ROW_SPAN_VALUE,
+                           "Something",
+                           "NSEW"),
+                          (top_level_frame_stub_i.TopLevelFrameStub(None),
+                           positioned_frame_i.SMALLEST_COLUM_AND_ROW_VALUE,
+                           positioned_frame_i.SMALLEST_COLUM_AND_ROW_VALUE,
+                           positioned_frame_i.SMALLEST_COLUM_SPAN_AND_ROW_SPAN_VALUE,
+                           positioned_frame_i.SMALLEST_COLUM_SPAN_AND_ROW_SPAN_VALUE,
+                           69)
```

Comment:
Is it ok to change the ordering of the elements half-way through?

Reply:
Nice

Reply:
what?